README.md

```diff
@@ -480,3 +496,32 @@ The documentation is available here: [documentation](https://pypsa-earth.readthe
     </td></tr>
 </table>
 <!-- readme: collaborators,contributors,restyled-commits/- -end -->
+
+## Licenses
```

Comment:
This section shall be moved here ideally:
https://github.com/pypsa-meets-earth/pypsa-earth/blob/main/doc/configtables/licenses.csv

This has been a misalignment with respect to the sec model I believe

Reply:
I moved it to the table :)